test/util/testfactory/utils.go

```diff
@@ -3,23 +3,31 @@ package testfactory
 import (
 	"context"
 	"encoding/hex"
+	"encoding/json"
+	"time"
 
+	"github.com/celestiaorg/celestia-app/app/encoding"
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/crypto/hd"
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
 	"github.com/cosmos/cosmos-sdk/simapp"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	tmrand "github.com/tendermint/tendermint/libs/rand"
 	rpctypes "github.com/tendermint/tendermint/rpc/core/types"
+	tmtime "github.com/tendermint/tendermint/types/time"
+	"google.golang.org/grpc"
 )
 
 const (
 	// nolint:lll
-	TestAccName  = "test-account"
-	TestAccMnemo = `ramp soldier connect gadget domain mutual staff unusual first midnight iron good deputy wage vehicle mutual spike unlock rocket delay hundred script tumble choose`
-	bondDenom    = "utia"
+	TestAccName               = "test-account"
+	TestAccMnemo              = `ramp soldier connect gadget domain mutual staff unusual first midnight iron good deputy wage vehicle mutual spike unlock rocket delay hundred script tumble choose`
+	bondDenom                 = "utia"
+	BaseAccountDefaultBalance = 10000
```

Comment:
why not use something large like in https://github.com/celestiaorg/celestia-app/blob/b5be4486cf1a62921ce4a329ed0305ee48d1316d/test/util/testfactory/utils.go#L89

is there a way to just use that function? or just use the standard mechanism in testnode? (where we create funded genesis accounts from the `testnode.Config.Accounts`?

https://github.com/celestiaorg/celestia-app/blob/b5be4486cf1a62921ce4a329ed0305ee48d1316d/test/util/testnode/full_node.go#L165

Reply:
We need to have a constant somewhere that can reference to it in our tests. something like `BaseAccountDefaultBalance` and it can be any value. We can change it to `99999999999999999`. We just need it to evaluate the balance for various tests.

Reply:
If we slightly change the standard mechanism function in testnode to adopt that constant, we can use it widely.